spec/routing/hq/dashboard_routing_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Hq::Dashboard routing', type: :routing, pending: true do
+
+  specify { expect(get: '/hq/dashboard'). to be_routable }
+
+end
+
+RSpec.describe "Hq::Dashboard redirecting", type: :request, pending: true do
+  before :each do
+    sign_in instance_double(AdminUser)
+  end
+
+  it "root redirects to dashboard" do
+    get '/hq'
+    expect(response).to redirect_to hq_dashboard_path
+  end
+end
```

Comment:
I see. This is not testing the `redirect` (/hq to /hq/dashboard) but it will remind the developer when it is working on dashboard.